Frameworks/Foundation/NSHTTPURLResponse.mm

```diff
@@ -85,7 +85,7 @@ - (id)initWithURL:(NSURL*)url statusCode:(NSInteger)statusCode HTTPVersion:(NSSt
 
         // Parse the filename from the Content-Disposition header field.
         NSString* contentDisposition = [allHeaderFields objectForKey:@"Content-Disposition"];
-        NSRange filenameTagPosition = [contentDisposition rangeOfString:@"filename=" options:NSBackwardsSearch];
```

Comment:
Got it.

---
In reply to: [73768171](https://github.com/Microsoft/WinObjC/pull/835#discussion_r73768171) [](ancestors = 73768171)